.travis.yml

```diff
@@ -6,4 +6,5 @@ go:
   - 1.7
   - 1.8
   - 1.9
+  - 1.10
```

Comment:
add `"` around, it's a `[known issue](https://github.com/travis-ci/travis-ci/issues/9247)

Reply:
TIL, updated.

Reply:
also, added `tip`.